index.html

```diff
@@ -30,6 +35,9 @@ <h3>
             target="_blank"
             >Get more info</a
           >
+          <!-- Example code:
+            <button type="button">Example</button>
+           -->
```

Comment:
Kindly, remove this comment, as it is not a good practice. It indicates that your code is not clean code.